pkg/kubectl/cmd/attach.go

```diff
@@ -114,9 +120,10 @@ type AttachOptions struct {
 
 	Pod *api.Pod
 
-	Attach    RemoteAttach
-	PodClient coreclient.PodsGetter
-	Config    *restclient.Config
+	Attach        RemoteAttach
+	PodClient     coreclient.PodsGetter
+	GetPodTimeOut time.Duration
```

Comment:
Make camel case concise with `GetPodRunningTimeoutFlag`, e.g. change it here to `GetPodTimeout`.